### PR TITLE
show silhouette icon where no avatar saved for user

### PR DESCRIPTION
**Pull request by justBanks**
New React component: [Avatar /] handles display of either the user's profile avatar or the generic Person icon.

New partial stylesheet.

I found three places to use the new component: ProjectOwnerCard, VolunteerCard, and AboutUserController.  Left the MainHeader untouched.

### CODE REVIEW

File: common/components/common/avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+type Props = {|
+  +user: UserAPIData,
+  +size: number
+|};
+
+class Avatar extends React.PureComponent<Props> {
+  render(): React$Node {
+    const user = this.props.user;
+    return (
+      user.user_thumbnail ?
```
Comment (marlonkeating) [inline common/components/common/avatar.jsx]:
```suggestion
      user.user_thumbnail
```

File: common/components/common/avatar.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <img className="upload_img upload_img_bdr VolunteerCard-img" src={user.user_thumbnail.publicUrl} /> :
```
Comment (marlonkeating) [inline common/components/common/avatar.jsx]:
```suggestion
        ? <img className="upload_img upload_img_bdr VolunteerCard-img" src={user.user_thumbnail.publicUrl} />
```

File: common/components/common/avatar.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <span className="Icon-container"><Person className="PersonIcon" style={{ fontSize: this.props.size }} /></span>
```
Comment (marlonkeating) [inline common/components/common/avatar.jsx]:
```suggestion
        : (<span className="Icon-container">
            <Person className="PersonIcon" style={{ fontSize: this.props.size }} />
        </span>)
```

Comment (justBanks) [reply common/components/common/avatar.jsx]: Alright, just cleaned up the code in a couple of places.

**Pull request merged**